content/doc/book/security/access-control/permissions.adoc

```diff
@@ -107,6 +105,101 @@ Anything they cannot accomplish through the existing UI they can do through the
 An instance on which an untrusted user gained _Overall/Administer_ permission should be considered fully compromised and should be replaced.
 All secrets (credentials, etc.) stored on that instance should be rotated or revoked and all artifacts created from that point on should be verified.
 
+
+=== Agent permissions
+
+==== Agent/Build
+
+This permission allows users to run jobs as them on agents.
```

Comment:
This is basically useless unless you already know what it does (it's related to https://www.jenkins.io/doc/book/security/build-authorization/ ).